### DISCUSSION TITLE
Can Xdrip for IOS read data from Libre Pro via Blucon Nightrider?

### DISCUSSION THREAD

**Discussion by zaiddroid** [JohanDegraeve/xdripswift#596] [Q&A]
Hello, I noticed that support for Libre Pro was added in May 2022 to the repository, however, it was only for miaomiao transmitters. 
I'd like to know if blucon nightrider works the same way as well. i tried but didn't get any reading. Just curious if there is a way to make this work. Please answer. Thanks

**Comment by paulplant** [comment]
This was already answered in the Facebook support group I think.

No way to add Libre Pro support via Blucon Nightrider to the app.
